token-metadata/js/idl/mpl_token_metadata.json

```diff
@@ -1165,13 +1166,15 @@
           "name": "useAuthorityRecord",
           "isMut": true,
           "isSigner": false,
-          "desc": "(Optional) Use Authority Record PDA If present the program Assumes a delegated use authority"
+          "desc": "Use Authority Record PDA If present the program Assumes a delegated use authority",
```

Comment:
Nit:

Use the same formatting as the previous description for clarity:

```
Use Authority Record PDA - If present, etc.
```

Reply:
This is generated from the contract description and I basically just removed `(Optional)` there.
I'm trying to keep the changes to a minimum trying to keep PRs as small as possible esp. to contract code.

We can fix those in a later PR as this one focuses solely on fixing optional accounts.